backend/tests/alert/test_alert.py

```diff
@@ -773,144 +1064,264 @@ def test_ignore_fields_phone_update(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.data["profile"]["phone"], "+12121234567")
         self.assertEqual(response.data["profile"]["email"], "example@email.com")
+        self.assertTrue(response.data["profile"]["push_notifications"])
         self.assertEqual(response.data["username"], "jacob")
         self.assertEqual(response.data["first_name"], "fname")
         self.assertEqual(response.data["last_name"], "lname")
         response = self.client.get(reverse("user-profile"))
         self.assertEqual(200, response.status_code)
         self.assertEqual(response.data["profile"]["phone"], "+12121234567")
         self.assertEqual(response.data["profile"]["email"], "example@email.com")
+        self.assertTrue(response.data["profile"]["push_notifications"])
+        self.assertEqual(response.data["username"], "jacob")
+        self.assertEqual(response.data["first_name"], "fname")
+        self.assertEqual(response.data["last_name"], "lname")
+
+    def test_ignore_fields_push_notifications_update(self):
```

Comment:
Why are we ignoring updates to the push settings? seems like users should be able to change that from the client?

Reply:
Sorry, I should really document these tests better.  This is testing the functionality that if the user only updates push_notifications, leaving out other fields, it ignores the other fields and only updates push_notifications.  I should really change the PCA PUT route to PATCH (have been meaning to do this)